cmd/create/ocmrole/cmd.go

```diff
@@ -0,0 +1,363 @@
+			ocm.Response: ocm.Success,
+		})
+		err = ocmrole.Cmd.RunE(cmd, []string{roleARN})
+		if err != nil {
+			reporter.Errorf("Unable to link role arn '%s' with the account id : '%s' : %v",
+				roleARN, currentAccount.ID(), err)
+		}
+	case aws.ModeManual:
+		ocmClient.LogEvent("ROSACreateOCMRoleModeManual", map[string]string{})
+		err = aws.GeneratePolicyFiles(reporter, env)
+		if err != nil {
+			reporter.Errorf("There was an error generating the policy files: %s", err)
+			ocmClient.LogEvent("ROSACreateOCMRoleModeManual", map[string]string{
+				ocm.Response: ocm.Failure,
+			})
+			os.Exit(1)
+		}
+		if reporter.IsTerminal() {
+			reporter.Infof("All policy files saved to the current directory")
+			reporter.Infof("Run the following commands to create the ocm role and policies:\n")
+		}
+		commands := buildCommands(prefix, permissionsBoundary, creator.AccountID, env)
+		fmt.Println(commands)
+	default:
+		reporter.Errorf("Invalid mode. Allowed values are %s", aws.Modes)
+		os.Exit(1)
+	}
+}
+
+func buildCommands(prefix string, permissionsBoundary string, accountID string, env string) string {
+	commands := []string{}
+	name := aws.GetRoleName(prefix, aws.OCMRole)
+	policyName := fmt.Sprintf("%s-Policy", name)
+	iamTags := fmt.Sprintf(
+		"Key=%s,Value=%s Key=%s,Value=%s Key=%s,Value=%s",
+		tags.RolePrefix, prefix,
+		tags.RoleType, aws.OCMRole,
+		tags.Environment, env,
+	)
+	permBoundaryFlag := ""
+	if permissionsBoundary != "" {
+		permBoundaryFlag = fmt.Sprintf("\t--permissions-boundary %s \\\n", permissionsBoundary)
+	}
+	createRole := fmt.Sprintf("aws iam create-role \\\n"+
+		"\t--role-name %s \\\n"+
+		"\t--assume-role-policy-document file://sts_%s_trust_policy.json \\\n"+
+		"%s"+
+		"\t--tags %s",
+		name, aws.OCMRolePolicyFile, permBoundaryFlag, iamTags)
+	createPolicy := fmt.Sprintf("aws iam create-policy \\\n"+
+		"\t--policy-name %s \\\n"+
+		"\t--policy-document file://sts_%s_permission_policy.json"+
+		"\t--tags %s",
+		policyName, aws.OCMRolePolicyFile, iamTags)
+	attachRolePolicy := fmt.Sprintf("aws iam attach-role-policy \\\n"+
+		"\t--role-name %s \\\n"+
+		"\t--policy-arn %s",
+		name, aws.GetPolicyARN(accountID, policyName))
+	commands = append(commands, createRole, createPolicy, attachRolePolicy)
+
+	return strings.Join(commands, "\n\n")
+}
+
+func createRoles(reporter *rprtr.Object, awsClient aws.Client,
+	prefix string, permissionsBoundary string, accountID string, env string) (string, error) {
+	roleName := aws.GetRoleName(prefix, aws.OCMRole)
+	policyARN := aws.GetPolicyARN(accountID, fmt.Sprintf("%s-Policy", roleName))
+	if !confirm.Prompt(true, "Create the '%s' role?", roleName) {
+		os.Exit(0)
+	}
+
+	filename := fmt.Sprintf("sts_%s_trust_policy.json", aws.OCMRolePolicyFile)
+	path := fmt.Sprintf("templates/policies/%s", filename)
+
+	policy, err := aws.ReadPolicyDocument(path, map[string]string{
+		"aws_account_id": aws.JumpAccounts[env],
+		"ocm_account_id": accountID,
```

Comment:
This is org id

Reply:
Should the label remain `ocm_account_id`? Just pass in the org account?

Reply:
Ideally the label is `ocm_org_id` or something like that, and send the AMS Organization ID (not the Account)

Reply:
although that's not a label, but a template variable to be replaced, so the name doesn't matter much